examples/computer_vision/shiftvit_pytorch/README.md

```diff
@@ -0,0 +1,66 @@
+# PyTorch ShiftViT Example
+
+This example demonstrates how to implement the [ShiftVit model](https://arxiv.org/abs/2201.10801) using Determined AI's
+PyTorch API. Model code is imported from the ShiftViT authors' [github implementation](https://github.com/microsoft/SPACH).
+
+## Files
+
+### Python
+
+* `model_def.py`: core code containing the Determined AI `PyTorchTrial` subclass.
+* `data.py`: handles the
+[MNIST](http://pytorch.org/vision/main/generated/torchvision.datasets.MNIST.html#torchvision.datasets.MNIST),
+[CIFAR10](http://pytorch.org/vision/main/generated/torchvision.datasets.CIFAR10.html),
+and [ImageNet](https://www.image-net.org) datasets.
+* `generate_blob_list.py`: scripts to pre-process GCS blobs (increases ImageNet speed)
```

Comment:
why can't the blob list be gathered and directly stored in memory at dataset creation instead of saving it to a file and then rereading it back in when creating the dataset?

Reply:
It's apparently much faster this way (per Corey), as getting the blob-list itself takes a while. So, just doing it once and storing the result is a helpful pre-processing step if you're going to use ImageNet more than once.